docs/UserGuide.md

```diff
@@ -1,37 +1,66 @@
 
-# Vendor Vault User Guide
+# VendorVault User Guide
 
-Vendor Vault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a  Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores. 
+VendorVault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a Command Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores. 
 
-<!-- * Table of Contents -->
-<page-nav-print />
+--- 
+
+### Table of Contents will be added here by Linkes
+
+
+---
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## How to get started
+
+1. If you are new to Vendor Vault, we suggest starting from the [Setting up VendorVault section](#setting-up-vendorvault).
+
+2. If you have already set up Vendor Vault and are looking for information of our commands and features, take a look at our [Features](#features) or [Command Summary](#command-summary) sections. 
+
+3. If you’re unsure about any of the terms used in this User Guide, please refer to the [Glossary](#glossary) section for definitions and explanations
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Setting up VendorVault 
+
+1. Java is the toolset that will allow you to use VendorVault on your system. Hence, first you need to ensure that you have Java `17` or above installed on your Computer. If you are unsure, you can check by following the steps below:
+   - 1a. Open up your command terminal. 
+     * Windows users can open the terminal by typing in `Command Prompt` in the search bar located at the bottom left of the screen. 
+     * Mac users can open the terminal by typing in `Terminal` using the spotlight search bar (accessed by pressing Command + Space).
+     * Linux users can open for the command terminal by searching for the `Terminal` or `Console` in the Applications Menu
+   
+   - 1b. Type in java -version to check if you have Java installed. If you have java installed you should see something like this:
+     ![Terminal Example showcasing java version](images/intro/terminalJavaVersion.png)
+   
+   - 1c. If you do not have Java installed, you can download it from [here](https://www.oracle.com/java/technologies/downloads/#java17?er=221886).
```

Comment:
Should we specify for mac users they need the specific AZUL 17 distribution?

Reply:
I dont this this this is needed correct?